modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -32,77 +32,154 @@ on {rh-openstack-first}
+* For the Ingress Controller backend service, ensure you manually define each node that runs the Ingress Controller in the external load balancer. For example, if the Ingress Controller moves to an undefined node, an connection outage can occur. 
 
-* On your load balancer, port 22623, which is used to serve ignition startup configurations to nodes, is not exposed outside of the cluster.
+.OpenShift API prerequisites
 
-* Your load balancer can access the required ports on each node in your cluster. You can ensure this level of access by completing the following actions:
-** The API load balancer can access ports 22623 and 6443 on the control plane nodes.
-** The ingress load balancer can access ports 443 and 80 on the nodes where the ingress pods are located.
+* You defined a front-end IP address.
+* You ensured that on your load balancer, TCP ports 6443 and 22623 are exposed on the front-end IP address. Check the following items:
+** Port 6443 provides access to the OpenShift API service.
+** Port 22623 can provide ignition startup configurations to nodes. 
+* The front-end IP address and port 6443 are reachable by all users of your system with a location external to your {product-title} cluster.
+* The front-end IP address and port 22623 are reachable only by {product-title} nodes.
+* The load balancer backend can communicate with {product-title} control plane nodes on port 6443 and 22623.
 
-ifdef::vsphere[]
-* Optional: If you are using multiple networks, you can create targets for every IP address in the network that can host nodes. This configuration can reduce the maintenance overhead of your cluster.
-endif::vsphere[]
+.Ingress Controller prerequisites
+
+* You defined a front-end IP address.
+* You ensured that on your load balancer, TCP ports 443 and 80 are exposed on the front-end IP address.
+* The front-end IP address, port 80 and port 443 are be reachable by all users of your system with a location external to your {product-title} cluster. 
+* The front-end IP address, port 80 and port 443 are reachable to all nodes that operate in your {product-title} cluster.
+* The load balancer backend can communicate with {product-title} nodes that run the Ingress Controller on ports 80, 443, and 1936.
 
 .Procedure
 
-. Enable access to the cluster from your load balancer on ports 6443, 443, and 80.
+. Configure health check resources for your backend services.
 +
-As an example, note this HAProxy configuration:
+.Example of a Kubernetes API server resource
 +
-.A section of a sample HAProxy configuration
+[source,yaml]
+----
+Path: HTTPS:6443/readyz
+Healthy threshold: 2
+Unhealthy threshold: 2
+Timeout: 10
+Interval: 10
+----
++
+.Example of a Machine config server resource
++
+[source,yaml]
+----
+Path: HTTPS:22623/healthz
+Healthy threshold: 2
+Unhealthy threshold: 2
+Timeout: 10
+Interval: 10
+----
++
+.Example of an Ingress Controller resource for HTTP and HTTPS traffic
```

Comment:
With my above comment, change this to "Ingress Controller Health Check Specification"